pkg/graveler/graveler.go

```diff
@@ -55,6 +55,14 @@ type RangeInfo struct {
 	Address string
 }
 
+type WriteCondition struct {
+	ifAbsent bool
+}
```

Comment:
The member `ifAbsent' is private and we like to control the value - I see that the only way to enable the value to true is using the `IfAbsent` function call. But it means that if I have a boolean which I would like to use when calling `Set`.  It will require an optional call to IfAbsent to get the right value for the call.